docs(gateway) add note on pg_sql and fips (#4467)
Kong/docs.konghq.com: app/gateway/changelog.md

  To enable FIPS mode, set [`fips`](/gateway/3.0.x/reference/configuration/#fips) to `on`.
  FIPS mode is only supported in Ubuntu 20.04.

  {:.note}

>   > **Note**: The Kong Gateway FIPS package is not compatible at this point with SSL 

Reviewer: ```suggestion
  > **Note**: The Kong Gateway FIPS package is not currently compatible with SSL 
```
